src/network/networkprotocol.h

```diff
@@ -925,6 +925,7 @@ enum CSMFlavourLimit : u64 {
 	CSM_FL_CHAT_MESSAGES = 0x00000002, // Disable chat message sending from CSM
 	CSM_FL_READ_ITEMDEFS = 0x00000004, // Disable itemdef lookups
 	CSM_FL_READ_NODEDEFS = 0x00000008, // Disable nodedef lookups
+	CSM_FL_LOAD_CLIENT_MODS = 0x000000010, // Disable mods provided by clients
```

Comment:
Having this as 16 means that if more byteflags are added, as is likely, then the 'disable client mod loading' byteflag will be in the middle with other byteflag either side, seems messy.
Seems better to have this byteflag as 0x00000001 and the others 0x00000002 and up. CSM flavours was added during 0.5.0dev so we can move these around.